app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -0,0 +1,383 @@
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Runtime Limit (Seconds)" helpKey="titus.deploy.runtimeLimitSecs">
+          <input
+            type="number"
+            className="form-control input-sm"
+            value={stage.cluster.runtimeLimitSecs}
+            onChange={e => this.stageFieldChanged('cluster.runtimeLimitSecs', e.target.value)}
+            min="1"
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Retries" helpKey="titus.deploy.retries">
+          <input
+            type="number"
+            className="form-control input-sm"
+            onChange={e => this.stageFieldChanged('cluster.retries', e.target.value)}
+            value={stage.cluster.retries}
+            min="0"
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Property File" helpKey="titus.deploy.propertyFile">
+          <input
+            type="text"
+            className="form-control input-sm"
+            onChange={e => this.stageFieldChanged('propertyFile', e.target.value)}
+            value={stage.propertyFile}
+          />
+        </StageConfigField>
+
+        <div className="form-group">
+          <div className="col-md-9 col-md-offset-1">
+            <div className="checkbox">
+              <label>
+                <input
+                  type="checkbox"
+                  checked={stage.showAdvancedOptions}
+                  onChange={e => this.stageFieldChanged('showAdvancedOptions', e.target.checked)}
+                />
+                <strong>Show Advanced Options</strong>
+              </label>
+            </div>
+          </div>
+        </div>
+
+        <div
+          className={`${stage.showAdvancedOptions !== true ? 'collapse' : ''} ${
+            stage.showAdvancedOptions === true ? 'collapse.in' : ''
+          }`}
+        >
+          <div className="form-group">
+            <label className="col-md-3 sm-label-right">
+              <span className="label-text">IAM Instance Profile (optional)</span>
+              <HelpField id="titus.deploy.iamProfile" />
+            </label>
+            <div className="col-md-4">
+              <input
+                type="text"
+                className="form-control input-sm"
+                value={stage.cluster.iamProfile}
+                onChange={e => this.stageFieldChanged('cluster.iamProfile', e.target.value)}
+              />
+            </div>
+            <div className="col-md-1 small" style={{ whiteSpace: 'nowrap', paddingLeft: '0px', paddingTop: '7px' }}>
+              in <AccountTag account={awsAccount} />
+            </div>
+          </div>
+          <StageConfigField label={firewallsLabel} helpKey="titus.job.securityGroups">
+            {(!stage.credentials || !stage.cluster.region) && (
+              <div>Account and region must be selected before {FirewallLabels.get('firewalls')} can be added</div>
```

Comment:
`firewallsLabel`

Reply:
Different casing, but made it less confusing.